### PR TITLE
Add last seen field

### LINKED ISSUE #61: Add date field

### PR DESCRIPTION
**Pull request by O-Darren**
Fixes #61

### CODE REVIEW

File: src/main/java/seedu/address/logic/parser/ParserUtil.java [hunk 1]
```diff
@@ -80,6 +81,21 @@ public static Address parseAddress(String address) throws ParseException {
...
+     *
+     * @throws ParseException if the given {@code lastSeen} is invalid.
+     */
+    public static LastSeen parseLastSeen(String lastSeen) throws ParseException {
+        requireNonNull(lastSeen);
+        String trimmedLastSeen = lastSeen.trim();
+        if (!LastSeen.isValidDate(trimmedLastSeen)) {
+            throw new ParseException(LastSeen.MESSAGE_CONSTRAINTS);
+        }
+        return new LastSeen(lastSeen);
```
Comment (IanCheah) [inline src/main/java/seedu/address/logic/parser/ParserUtil.java:96]: Hi @O-Darren,  just to double confirm if you intended to `return new LastSeen(lastSeen)` or was it supposed to be `return new LastSeen(trimmedLastSeen)`

Comment (O-Darren) [reply src/main/java/seedu/address/logic/parser/ParserUtil.java:96]: Intended to return trimmedLastSeen, thanks for the catch

File: src/test/java/seedu/address/model/person/LastSeenTest.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+import org.junit.jupiter.api.Test;
+
+public class LastSeenTest {
+
+    @Test
+    public void constructor_null_throwsNullPointerException() {
+        assertThrows(NullPointerException.class, () -> new LastSeen(null));
+    }
+
+    @Test
```
Comment (nigellqx) [inline src/test/java/seedu/address/model/person/LastSeenTest.java:16]: @O-Darren do you mean lastseen?

Comment (O-Darren) [reply src/test/java/seedu/address/model/person/LastSeenTest.java:16]: Yep, fixed! Thank you!

**Pull request merged**